app/components/sections/route-plan-blueprints/route-plan-blueprint-table/component.js

```diff
@@ -0,0 +1,64 @@
+import Ember from "ember";
+import computed from "ember-computed-decorators";
+import TemplateValidations from "last-strawberry/validators/route-plan-blueprint";
+
+export default Ember.Component.extend({
+  session: Ember.inject.service(),
+
+  classNames: "col stretch",
+
+  filterTerm: "",
+
+  hasMatch(routePlanBlueprint, query) {
+    const reg = new RegExp(query, "i");
+    return reg.test(routePlanBlueprint.get("name"));
+  },
+
+  @computed("routePlanBlueprints.[]", "filterTerm")
+  filteredItems(routePlanBlueprints, query){
+    return routePlanBlueprints
+      .filter(i => this.hasMatch(i, query));
+  },
+
+  @computed("session")
+  validators(session) {
+    return TemplateValidations(session);
+  },
+
+  @computed("users.[]")
```

Comment:
We should add the .@each.{name} key in case name ever changes this won't update.